EmojiComponents/src/main/java/com/infomaniak/emojicomponents/views/EmojiReactionsView.kt

```diff
@@ -62,12 +65,16 @@ class EmojiReactionsView @JvmOverloads constructor(
     private var addReactionDisabledColor: Int? = null
 
     init {
+        val composeView = ComposeView(context)
+        composeView.setContent { ComposeViewContent() }
+        addView(composeView)
```

Comment:
I think you should end the init with these lines. 
It's both more "logical" to add the view in last, and setViewCompositionStrategy should be declared before setContent from what I understand. 
Because the strategy seems to be applicated when setContent is called. (At least in the doc, it is always called before setContent)